validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkSubmissionValidatorTest.java

```diff
@@ -178,7 +194,7 @@ public void setUpLocalTests() {
                                                 errorMessagesCache,
                                                 mockIBulkSubmissionDao,
                                                 requestTypeXmlNodeValidator,
-                                                new ConcurrentHashMap());
+                                                new ConcurrentHashMap<>());
```

Comment:
we normally don't need <>, we can use this new ConcurrentHashMap()

Reply:
Removing the <> causes Sonar issue S3740 to be flagged:

> **Raw types should not be used**
> Generic types shouldn’t be used raw (without type parameters) in variable declarations or return values. Doing so bypasses generic type checking, and defers the catch of unsafe code to runtime.

This is classified as a major code smell.

It appears to be picking up that the parameter it's being passed to is declared as `Map<String, IInFlightMessage>`.